roles/openshift_master/tasks/main.yml

```diff
@@ -87,6 +87,9 @@
       master_count: "{{ openshift_master_count | default(None) }}"
       controller_lease_ttl: "{{ osm_controller_lease_ttl | default(None) }}"
       master_image: "{{ osm_image | default(None) }}"
+    role: common
+    local_facts:
+      imageconfig_latest: "{{ openshift_imageconfig_latest | default(False) }}"
```

Comment:
For setting this, we should also be using the bool filter to ensure that we can accept all of the ansible true/false values and that we do not end up with errors around the value being a string:
```
imageconfig_latest: "{{ openshift_imageconfig_latest | default(False) | bool }}"
```